src/main/java/net/malisis/core/client/gui/component/interaction/UITextField.java

```diff
@@ -1089,10 +1089,7 @@ public String getPropertyString() {
     }
 
     @Subscribe
-    public void onSizeUpdate(SizeChangeEvent<UIComponent> event) {
-        // if (event.getComponent() != this)
-        // buildLines();
-    }
+    public void onSizeUpdate(SizeChangeEvent<UIComponent> event) {}
```

Comment:
Is there a point inkeeping an empty method subscribed to an event?

Reply:
maybe it can be mixined i guess

Reply:
The `@Subscribe` can be commented out with the note that one needs to uncomment it if they want for the method to do its job. What do you think about this approach, @leagris?

Reply:
> The `@Subscribe` can be commented out with the note that one needs to uncomment it if they want for the method to do its job. What do you think about this approach, @leagris?

There is some bad code design smell here I think. But a refactor would be out of scope and there is probably a lot more at other places.

To acknowledge it needs attention while preserving current operation, you may leave a TODO comment.

```java
/*
  * TODO: Need reconsider registration of this empty method to the event listener.
  * It has been spotted during cleanup of commented-out code.
  * This is kept as-is to preserve code operation and because further code change would be
  * out of scope of this automated cleanup.
  */
```

Reply:
https://github.com/GTNewHorizons/MalisisCore/pull/5/commits/6e550133b554cf911e546294aec9cd1da3381d67
@leagris please let me know if this approach is acceptable